src/routes/(console)/organization-[organization]/usage/[[invoice]]/+page.svelte

```diff
@@ -21,32 +21,39 @@
     import { formatCurrency, formatNumberWithCommas } from '$lib/helpers/numbers';
     import { Icon, Layout, Link, Tooltip, Typography } from '@appwrite.io/pink-svelte';
     import { IconChartSquareBar, IconInfo } from '@appwrite.io/pink-icons-svelte';
+    import { onMount } from 'svelte';
+    import type { UsageProjectInfo } from '../../store';
 
     export let data;
 
     const tier = data?.plan
         ? (data.plan.$id as Tier)
         : (data?.currentInvoice?.plan ?? $organization?.billingPlan);
+
     const plan = data?.plan ?? undefined;
 
     $: projects = (data.organizationUsage as OrganizationUsage).projects;
 
+    let usageProjects: { [key: string]: UsageProjectInfo } = {};
```

Comment:
```suggestion
    let usageProjects: Record<string, UsageProjectInfo> = {};
```
can be a cleaner approach for the type 👍🏻